libvcx/build.rs

```diff
@@ -80,46 +79,25 @@ fn main() {
             }
         };
 
-        println!("cargo:rustc-link-search=native={}",libindy_lib_path);
+        println!("cargo:rustc-link-search=native={}", libindy_lib_path);
         println!("cargo:rustc-link-lib=static=indy");
+
         println!("cargo:rustc-link-search=native={}", openssl);
         println!("cargo:rustc-link-lib=static=crypto");
         println!("cargo:rustc-link-lib=static=ssl");
-
-    }else if target.contains("darwin"){
+    } else if target.contains("darwin") {
         //OSX specific logic
         println!("cargo:rustc-link-lib=sodium");
         println!("cargo:rustc-link-lib=zmq");
         println!("cargo:rustc-link-lib=indy");
         //OSX does not allow 3rd party libs to be installed in /usr/lib. Instead install it in /usr/local/lib
         println!("cargo:rustc-link-search=native=/usr/local/lib");
-    }else if target.contains("-linux-"){
+    } else if target.contains("-linux-") {
         //Linux specific logic
         println!("cargo:rustc-link-lib=indy");
         println!("cargo:rustc-link-search=native=/usr/lib");
-    }else if target.contains("-windows-") {
-        println!("cargo:rustc-link-lib=indy.dll");
-
-        let profile = env::var("PROFILE").unwrap();
-        println!("profile={}", profile);
-
-        let output_dir = env::var("OUT_DIR").unwrap();
-        println!("output_dir={}", output_dir);
-        let output_dir = Path::new(output_dir.as_str());
-
-        let indy_dir = env::var("INDY_DIR").unwrap_or(format!("..\\..\\libindy\\target\\{}", profile));
-        println!("indy_dir={}", indy_dir);
-        let indy_dir = Path::new(indy_dir.as_str());
-
-        let dst = output_dir.join("..\\..\\..\\..");
-        println!("cargo:rustc-flags=-L {}", indy_dir.as_os_str().to_str().unwrap());
-
-        let files = vec!["indy.dll", "libeay32md.dll", "libsodium.dll", "libzmq.dll", "ssleay32md.dll"];
-        for f in files.iter() {
-            if let Ok(_) = fs::copy(&indy_dir.join(f), &dst.join(f)) {
-                println!("copy {} -> {}", &indy_dir.join(f).display(), &dst.join(f).display());
-            }
-        }
+    } else if target.contains("-windows-") {
+        panic!("Windows not supported.");
```

Comment:
I don't know if vcx builds work on windows because I don't think we officially support it, so I removed it as a part of build cleanup. 
Well it doesn't to any harm to keep, but I am agnostic to this - we can keep this code if you think we should keep it.

Reply:
I would keep it. We don't know if it works, but in the case it does, we would be panicking for no reason.